Documentation/operations/system_requirements.rst

```diff
@@ -81,22 +83,80 @@ RancherOS_                 >= 1.5.5
 .. _Opensuse: https://www.opensuse.org/
 .. _RancherOS: https://rancher.com/rancher-os/
 
-.. [#centos_foot] CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
-    whereas CentOS 8 ships with a supported kernel. Note that some more advanced features may not be available on CentOS 7 even with a third-party kernel. For full details on which kernel config options must be enabled in order to use various features, see the section on :ref:`admin_kernel_version` requirements below.
-
 .. note:: The above list is based on feedback by users. If you find an unlisted
           Linux distribution that works well, please let us know by opening a
           GitHub issue or by creating a pull request that updates this guide.
 
-.. note:: Systemd 245 and above (``systemctl --version``) overrides ``rp_filter`` setting
-          of Cilium network interfaces. This introduces connectivity issues
-          (see :gh-issue:`10645` for details). To avoid that, configure
-          ``rp_filter`` in systemd using the following commands:
 
-          .. code-block:: shell-session
+CentOS 7
+~~~~~~~~
+
+CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
+whereas CentOS 8 ships with a supported kernel. Note that some more advanced
+features may not be available on CentOS 7 even with a third-party kernel. For
+full details on which kernel config options must be enabled in order to use
+various features, see the section on :ref:`admin_kernel_version` requirements
+below.
+
+
+``systemd``-based distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``systemd`` 245 and above overrides the ``rp_filter`` setting of Cilium network
+interfaces. This introduces connectivity issues (see :gh-issue:`10645` for
+details). To avoid that, configure ``rp_filter`` in ``systemd`` using a command
+such as the following:
+
+.. code-block:: shell-session
+
+    echo 'net.ipv4.conf.lxc*.rp_filter = 0' > /etc/sysctl.d/99-override_cilium_rp_filter.conf
+    systemctl restart systemd-sysctl
+
+Keep in mind, however, that some distributions may require additional
+overrides. For example, in Ubuntu 22.04 it's also necessary to edit
+``/etc/sysctl.d/10-network-security.conf``, otherwise this change will get
+undone over time. Please refer to your distribution's documentation for
+the right way to perform this override.
```

Comment:
This should no longer be necessary as we have it fixed on all Cilium stable versions https://github.com/cilium/cilium/pull/20072